packages/block-editor/src/components/block-tools/index.js

```diff
@@ -92,19 +92,31 @@ export default function BlockTools( {
 			return;
 		}
 
-		if ( isMatch( 'core/block-editor/move-up', event ) ) {
+		if (
+			isMatch( 'core/block-editor/move-up', event ) ||
+			isMatch( 'core/block-editor/move-down', event )
+		) {
 			const clientIds = getSelectedBlockClientIds();
 			if ( clientIds.length ) {
 				event.preventDefault();
 				const rootClientId = getBlockRootClientId( clientIds[ 0 ] );
-				moveBlocksUp( clientIds, rootClientId );
-			}
-		} else if ( isMatch( 'core/block-editor/move-down', event ) ) {
-			const clientIds = getSelectedBlockClientIds();
-			if ( clientIds.length ) {
-				event.preventDefault();
-				const rootClientId = getBlockRootClientId( clientIds[ 0 ] );
-				moveBlocksDown( clientIds, rootClientId );
+				const direction = isMatch( 'core/block-editor/move-up', event )
+					? 'up'
+					: 'down';
+				if ( direction === 'up' ) {
+					moveBlocksUp( clientIds, rootClientId );
+				} else {
+					moveBlocksDown( clientIds, rootClientId );
+				}
+				const blockLength = Array.isArray( clientIds )
+					? clientIds.length
+					: 1;
+				const message = sprintf(
+					// translators: %s: term "block" in either singular or plural form.
+					_n( '%s moved.', '%s moved.', blockLength ),
+					blockLength > 1 ? __( 'Blocks' ) : __( 'Block' )
+				);
```

Comment:
Sorry for the insufficient explanation. In this implementation, we cannot know how many blocks have actually been moved.

What I wanted to say is to tell how many blocks have been moved. That is, the code I proposed in [this comment](https://github.com/WordPress/gutenberg/pull/64966#issuecomment-2443058839).

I think this code is consistent with the implementation below:

https://github.com/WordPress/gutenberg/blob/986fc5519b9455c2683f519abe4d6446111d5e38/packages/block-editor/src/components/inserter/hooks/use-insertion-point.js#L183-L187

https://github.com/WordPress/gutenberg/blob/986fc5519b9455c2683f519abe4d6446111d5e38/packages/editor/src/components/block-manager/index.js#L87-L95

Reply:
@t-hamano Added more consistent messaging